.env.example

```diff
@@ -29,20 +29,20 @@ BBN_FINALITY_GADGET_RPC=
 ## mnemonic "test test test test test test test test test test test junk" 
 
 # Admin address
-GS_ADMIN_ADDRESS=0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
-GS_ADMIN_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+GS_ADMIN_ADDRESS=
+GS_ADMIN_PRIVATE_KEY=
```

Comment:
i'd still keep those example values or at least the 0x prefix

Reply:
i removed those values as:
- the current script will backup any existing keys and transfer out any balances
- it could be misleading to have existing values there since those are placeholder keys

perhaps we can add back inline comments like so:

```bash
# OP Batcher contract address
# e.g. GS_BATCHER_ADDRESS=0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
GS_BATCHER_ADDRESS=
```

Reply:
got it sure